server/routes/adjudicationForReport/prisonerReport/prisonerReportPage.ts

```diff
@@ -171,6 +171,12 @@ export default class prisonerReportRoutes {
       user
     )
 
+    const hideReportNumberAndPrintStatuses = [
```

Comment:
Minor one - could name this `hideReportNumberAndPrintForAdjudicationStatuses` to be more explicit